openmdao/docs/openmdao_book/features/reports/reports_system.ipynb

```diff
@@ -92,8 +92,20 @@
    "source": [
     "## Setting the directory for the reports\n",
     "\n",
-    "If the user wishes to have the reports directory placed into a different directory other than the default current working directory, the user can set the environment variable, `OPENMDAO_REPORTS_DIR` to the name of that directory. The directory does not have to exist beforehand. It will be created by the reporting system. As an example, if `OPENMDAO_REPORTS_DIR` is set to \"my_reports\", then, if the `Problem` name is `problem1`, then the reports will be written to the \"my_reports/problem1\" directory.\n",
+    "If the user wishes to have the reports directory placed into a different directory other than the default current working directory, the user has two options for doing so.\n",
     "\n",
+    "First, this option can be set programmatically using the `set_reports_dir` function from `openmdao.api`.\n",
+    "Note that this is a global OpenMDAO setting and therefore it cannot take different values for different processors if running multiple problems simultaneously under MPI.\n",
+    "\n",
+    "The default value may be set using the environment variable `OPENMDAO_REPORTS_DIR` to the name of that directory. The directory does not have to exist beforehand. It will be created by the reporting system. As an example, if `OPENMDAO_REPORTS_DIR` is set to \"my_reports\", then, if the `Problem` name is `problem1`, then the reports will be written to the \"my_reports/problem1\" directory.\n",
```

Comment:
I think you should either bulletize the two options, or start the second sentence with "Alternatively" or "The second option" etc to more clearly demarcate the two options..